### ISSUE TITLE
Issues with Lasso for SPH-model

### ISSUE THREAD

**Issue by MzHf0** [lasso-gmbh/lasso-python#17]
Thank you a lot for this great Tool!

At the moment I am using Lasso for the analysis of a sph-model:

- Is there a FilterType for sph?

- When reading the d3plot file I get the warning:

> WARNING:lasso.dyna.D3plot:A failure in _read_states_sph, pressure was caught:
Traceback (most recent call last):
  File "c:\***\D3plot.py", line 4021, in _read_states_sph
    array_dict[ArrayType.sph_stress] = sph_data[:, i_var:i_var + 6]\
ValueError: cannot reshape array of size 6 into shape (1,40748,6)

The same warning repeats for strain and strainrate

- The d3plot.arrays['sph_history_vars'] isnt read, or at least is not part of the d3plot.arrays.keys 

Can you help me with that?
Thank you.

**Comment by tnt-codie** [comment]
Seems that there were errors while reading the SPH data, Do you have a model to share?

**Comment by MzHf0** [comment]
Yes I can send you some files. But not publicly. Where should I send them?

**Comment by codie3611** [comment]
You can send it to constantin.diez@gmail.com.
